<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ososov</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="FIOLabel">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>40</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ФИО</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="FIOLineEdit">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>40</y>
      <width>108</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="VacancyLineEdit">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>80</y>
      <width>108</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateEdit" name="BirthDayDateEdit">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>120</y>
      <width>96</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="VacancyLabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>80</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Должность</string>
    </property>
   </widget>
   <widget class="QLabel" name="BirthDayLabel">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>120</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Дата Рождения</string>
    </property>
   </widget>
   <widget class="QLabel" name="PhotoLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>15</y>
      <width>131</width>
      <height>141</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string>Фото</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DownloadPushButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Загрузить</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="ManRadioButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>200</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>М</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="WomanRadioButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>200</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Ж</string>
    </property>
   </widget>
   <widget class="QPushButton" name="OKPushButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>360</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Готово</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="PathToImageLineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
